### PR TITLE
Feature expose policies

### PR DESCRIPTION
**Pull request by goruha**
## What
* Output policies name / ARN

## Why
* Allow caller to attach policies to necessary principals

### CODE REVIEW

File: output.tf [hunk 1]
```diff
@@ -1,15 +1,54 @@
...
 output "repository_name" {
-  value = "${aws_ecr_repository.default.name}"
+  value       = "${aws_ecr_repository.default.name}"
+  description = "Registry name"
 }
 
 output "role_name" {
-  value = "${join("", aws_iam_role.default.*.name)}"
+  value       = "${join("", aws_iam_role.default.*.name)}"
+  description = "Assume Role name to get access registry"
```
Comment (Jamie-BitFlight) [inline output.tf]: Assume Role name to get registry access

File: output.tf [hunk 2]
```diff
@@ -1,15 +1,54 @@
...
+}
+
+output "role_arn" {
+  value       = "${join("", aws_iam_role.default.*.arn)}"
+  description = "Assume Role ARN to get access registry"
```
Comment (Jamie-BitFlight) [inline output.tf]: Assume Role ARN to get registry access

File: output.tf [hunk 3]
```diff
@@ -1,15 +1,54 @@
...
+}
+
+output "policy_login_name" {
+  value       = "${aws_iam_policy.login.name}"
+  description = "IAM Policy name allow to login in ECR"
```
Comment (Jamie-BitFlight) [inline output.tf]: The IAM Policy name to be given access to login in ECR

File: output.tf [hunk 4]
```diff
@@ -1,15 +1,54 @@
...
+}
+
+output "policy_login_arn" {
+  value       = "${aws_iam_policy.login.arn}"
+  description = "IAM Policy ARN allow to login in ECR"
```
Comment (Jamie-BitFlight) [inline output.tf]: The IAM Policy ARN to be given access to login in ECR

File: output.tf [hunk 5]
```diff
@@ -1,15 +1,54 @@
...
+}
+
+output "policy_read_name" {
+  value       = "${aws_iam_policy.read.name}"
+  description = "IAM Policy name allow to pull images from ECR"
```
Comment (Jamie-BitFlight) [inline output.tf]: "The IAM Policy name to be given access to pull images from ECR"

File: output.tf [hunk 6]
```diff
@@ -1,15 +1,54 @@
...
+}
+
+output "policy_read_arn" {
+  value       = "${aws_iam_policy.read.arn}"
+  description = "IAM Policy ARN allow to pull images from ECR"
```
Comment (Jamie-BitFlight) [inline output.tf]: "The IAM Policy ARN to be given access to pull images from ECR"

File: output.tf [hunk 7]
```diff
@@ -1,15 +1,54 @@
...
+}
+
+output "policy_write_name" {
+  value       = "${aws_iam_policy.write.name}"
+  description = "IAM Policy name allow to push images to ECR"
```
Comment (Jamie-BitFlight) [inline output.tf]: "The IAM Policy name to be given access to push images to ECR"

File: output.tf [hunk 8]
```diff
@@ -1,15 +1,54 @@
...
+}
+
+output "policy_write_arn" {
+  value       = "${aws_iam_policy.write.arn}"
+  description = "IAM Policy ARN allow to push images to ECR"
```
Comment (Jamie-BitFlight) [inline output.tf]: "The IAM Policy ARN to be given access to push images to ECR"

File: variables.tf [hunk 9]
```diff
@@ -1,8 +1,14 @@
...
+  description = "Namespace (e.g. `cp` or `cloudposse`)   "
+}
 
-variable "stage" {}
+variable "stage" {
+  description = "Stage (e.g. `prod`, `dev`, `staging`) "
+}
 
 variable "roles" {
   type        = "list"
```
Comment (Jamie-BitFlight) [inline variables.tf]: Description missing

**Comment by goruha** [comment]
@Jamie-BitFlight Done

File: README.md [hunk 10]
```diff
@@ -31,6 +31,35 @@ module "ecr" {
 }
 ```
 
+Example of attaching polices to user (for CI/CD purpose)
```
Comment (sarkis) [inline README.md]: Example of attaching policies to a user for CI/CD

File: variables.tf [hunk 11]
```diff
@@ -1,8 +1,14 @@
-variable "name" {}
+variable "name" {
+  description = "The Name of the application or solution  (e.g. `bastion` or `portal`)"
+}
 
-variable "namespace" {}
+variable "namespace" {
+  description = "Namespace (e.g. `cp` or `cloudposse`)   "
```
Comment (sarkis) [inline variables.tf]: extra spaces between ) and "

File: variables.tf [hunk 12]
```diff
@@ -1,8 +1,14 @@
...
+}
 
-variable "stage" {}
+variable "stage" {
+  description = "Stage (e.g. `prod`, `dev`, `staging`) "
```
Comment (sarkis) [inline variables.tf]: extra space between ) and "

File: variables.tf [hunk 13]
```diff
@@ -11,18 +17,21 @@ variable "roles" {
...
+  default     = []
+  description = "Additional attributes (e.g. `policy` or `role`)"
 }
 
 variable "tags" {
-  type    = "map"
-  default = {}
+  type        = "map"
+  default     = {}
+  description = "Additional tags (e.g. `map('BusinessUnit`,`XYZ`)"
```
Comment (sarkis) [inline variables.tf]: bad formatting around map('Business...

**Pull request merged**